window.go

```diff
@@ -519,3 +521,11 @@ func (w *Window) Unmaximize() (err error) {
 	_, err = synchronousEvent(w.ctx, w, w.w, Event{Name: EventNameWindowCmdUnmaximize, TargetID: w.id}, EventNameWindowEventUnmaximize)
 	return
 }
+
+func (w *Window) ExecuteJavaScript(code string) (err error) {
```

Comment:
Could you add a test as well? (you can copy the one for unmaximize for instance)